modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -6,82 +6,53 @@
 ====
 
-=== Using the web console to opt in to automatic CSI migration
-
-==== Prerequisites
-
-* Access to the {product-title} web console.
-
-* Access to the cluster with cluster-admin privileges.
-
-===== Procedure
-
-To opt in to automatic CSI migration for vSphere:
-
-. Log in to the web console.
-
-. Click *Administration* -> *CustomResourceDefinitions*.
-
-. On the *CustomResourceDefinitions* page, type "Storage" to find the `Storage` custom resource (CR).
-
-. Click the *Storage* CR.
+== New installations of {product-title}
+For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
 
-. On the *storages.operator.openshift.io* page, click the *Instances* tab.
+[id="updating-openshift_from_4.13_{context}"]
+== Updating from {product-title} 4.13 to 4.14
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.13 to 4.14, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} update is blocked. After updating vSphere, your {product-title} update can occur and automatic migration is enabled by default.
 
-. Click the name of the desired instance, and then click the *YAML* tab.
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} update by performing an administrator acknowledgment:
 
-. Set the `spec.vsphereStorageDriver` parameter to `CSIWithMigrationDriver`, as shown in the following example:
-+
-[source, yaml]
+[source, cli]
 ----
-....
-spec:
-  logLevel: Normal
-  managementState: Managed
-  operatorLogLevel: Normal
-  vsphereStorageDriver: CSIWithMigrationDriver <1>
-...
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.13-kube-1.27-vsphere-migration-in-4.14":"true"}}' --type=merge
 ----
-<1> `spec.vsphereStorageDriver` parameter set to `CSIWithMigrationDriver`
 
-. Click *Save*.
-
-=== Using the CLI to opt in to automatic CSI migration
-
-==== Prerequisites
+[IMPORTANT]
+====
+If you do *not* update to vSphere 7.0 Update 3L or 8.0 Update 2 and use an administrator acknowledgment to update to {product-title} 4.14, known issues can occur. link:https://access.redhat.com/node/7011683[Before proceeding with the administrator acknowledgement, carefully read this knowledge base article].
+====
 
-* Access to the cluster with cluster-admin privileges.
+[id="updating-openshift_from_4.12_{context}"]
+== Updating from {product-title} 4.12 to 4.14
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.12 to 4.14, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} update is blocked. After updating vSphere, your {product-title} update can occur and automatic migration is enabled by default.
 
-==== Procedure
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} update by performing an administrator acknowledgment:
```

Comment:
since there are two admin-ack to perform, maybe rephrase the sentence to make it clear that the two acks are required

Reply:
Done.